courses/views/v1/__init__.py

```diff
@@ -190,3 +204,31 @@ def destroy(self, request, *args, **kwargs):
         if deactivated_enrollment is None:
             return Response(status=status.HTTP_400_BAD_REQUEST)
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+    def partial_update(self, request, *args, **kwargs):
```

Comment:
Yeah, I know that but that API PATCH request is not being used anywhere else in the code. So, that's why it didn't matter to me and even if you see edX API it is designed that why if you don't pass `receive_emails` it will consider it to mark for unsubscription. So, while developing that I had pretty much thing in mind. 

I guess that could be skipped, for now, we will see once we will encounter other uses of that API (PATCH) version.

Let me know what's your opinion.

Reply:
I think we can apply a simple fix, run the subscription-related code only if `receive_emails` key was passed it can be `True/False` but not None. Is that feasible?

Reply:
sure